pyjsx/codecs.py

```diff
@@ -1,11 +1,19 @@
+from __future__ import annotations  # noqa: A005
```

Comment:
Checked it out locally and this does not need a noqa:
```suggestion
from __future__ import annotations
```